### PR TITLE
Ma premiere branche

### PR DESCRIPTION
**Pull request by piecharton**
Voici la première pull request du projet !

Added prompt displaying minishell>
Stops when using Ctrl D

### CODE REVIEW

File: example.c [hunk 1]
```diff
@@ -6,13 +6,16 @@
...
 
 #include "../includes/example.h"
 
 int	ft_max(int a, int b)
 {
-	return ((a > b) ? a : b);
+	if (a > b)
+		return (a);
+	else
+		return (b);
```
Comment (Crysicia) [inline example.c:20]: You can remove the `else` keyword here

File: lexer.c [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+{
+	char	*line;
+	int		ret;
+
+	ret = get_next_line(0, &line);
+	if (ret == -1)
+	{
+		puts("gnl error");
+		exit(-1);
+	}
```
Comment (Crysicia) [inline lexer.c:26]: There is a potential memory leak here.
GNL's line remainder could (And will) still be malloc'ed

**Review by Crysicia — CHANGES_REQUESTED** [review]

File: example.c (hunk 1 above)
Comment (piecharton) [reply example.c:20]: I find it esthetically pleasing

File: lexer.c (hunk 2 above)
Comment (piecharton) [reply lexer.c:26]: True. I did not plan any error managment yet. Will take care of.

File: lexer.c [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+int	lexer(char *line)
+{
+	char	*tmp;
+	int		i;
+
+	i = 0;
+	while (*line)
+	{
+		tmp = get_word(&line);
```
Comment (piecharton) [inline lexer.c:40]: Get_word returns a char* but this one is just displayed on the screen atm.
I either need to store this char * in a struct or transform it

**Comment by Crysicia** [comment]
Feel free to merge whenever you want :)

**Pull request merged**